### PR TITLE
Xsdk compliance doc

### PR DESCRIPTION
**Pull request by ashermancinelli**
Introduces xSDK compliance doc and updates according to #99.

### CODE REVIEW

**Comment by ashermancinelli** [comment]
@cnpetra Could you give this a look over and merge if you're happy? Having at least a draft of the document in our development branches will help move along the xsdk work.

File: doc/hiop-policy-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+|**M2.** Provide a comprehensive test suite for correctness of installation verification. |Full| HiOp contains several test drivers for integration tests as well as about 240 unit tests for each key linear algebra kernel.|
+|**M3.** Employ user-provided MPI communicator (no MPI_COMM_WORLD). Don't assume a full MPI 3 implementation without checking. Provide an option to prevent any changes to MPI error-handling if it is changed by default. |Full| HiOp objects take an MPI communicator in the constructor. |
+|**M4.** Give best effort at portability to key architectures (standard Linux distributions, GNU, Clang, vendor compilers, and target machines at ALCF, NERSC, OLCF). |Full| Our continuous integration tests every branch on multiple platforms. |
+|**M5.** Provide a documented, reliable way to contact the development team. |Full| [Submit issues here](https://github.com/LLNL/hiop/issues). |
+|**M6.** Respect system resources and settings made by other previously called packages (e.g. signal handling). |Full| None. |
+|**M7.** Come with an open source (BSD style) license. |Full| Use 2-clause BSD license. |
```
Comment (cnpetra) [inline doc/hiop-policy-compatibility.md:21]: I think it HiOp has the 3-clause BSD

**Review by cnpetra — APPROVED** [review]

File: doc/hiop-policy-compatibility.md (hunk 1 above)
Comment (ashermancinelli) [reply doc/hiop-policy-compatibility.md]: Resolved

**Review by pelesh — APPROVED** [review]

File: doc/hiop-policy-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# xSDK Community Policy Compatibility for HiOp
+
+This document summarizes the efforts of current and future xSDK member packages to achieve compatibility with the xSDK community policies. Below only short descriptions of each policy are provided. The full description is available [here](https://github.com/xsdk-project/xsdk-community-policies)
+and should be considered when filling out this form.
+
+Please, provide information on your compability status for each mandatory policy, and if possible also for recommended policies.
```
Comment (bl-ue) [inline doc/hiop-policy-compatibility.md:6]: Hello, I was reading this document (I'm a silent viewer of these repos) and I found just one typo :)
```suggestion
Please, provide information on your compatibility status for each mandatory policy, and if possible also for recommended policies.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: doc/hiop-policy-compatibility.md (hunk 2 above)
Comment (cnpetra) [reply doc/hiop-policy-compatibility.md]: thanks

**Pull request merged**